extension.js

```diff
@@ -34,6 +34,11 @@ function activate(context) {
       }
 
       vscode.workspace.findFiles(fileToOpen, "**/.elixir_ls/**").then(files => {
+        
+        if(!files.length){
+          return vscode.window.showInformationMessage("There's not matching tests");
```

Comment:
```suggestion
          return vscode.window.showInformationMessage(`There isn't a test file matching "${fileName}.ex"`);
```

Just change this and we are good to go